### PR TITLE
Modular

### PR DESCRIPTION
**Pull request by trevorackerman**
Hi All,

I'm proposing to 
- switch to using a more 'just bootstrap' styling
- compactify the navigation controls into a navbar header so more of the map is visible
- Use Javascript modules for separating the concerns of the map, the layers, and the trail data
- Also added javascript sourcemap for the webpack config to ease debugging

Check it out! :)

I've included a screenshot on how it's rendering in Chrome on my Mac laptop
trailsy-screenshot

### CODE REVIEW

**Comment by garborg** [comment]
@trevorackerman This looks great! Perhaps add `bundle.js.map` to `.gitignore` like `bundle.js` is and remove it from the PR?

File: package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "webpack-dev-server": "^1.9.0"
   },
   "scripts": {
-    "start": "webpack-dev-server --progress --colors",
+    "start": "webpack-dev-server --progress --colors --port 9000",
+    "sourcemap": "webpack --dev-tool source-map",
     "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (garborg) [inline package.json:23]: I'm not sure if 'sourcemap' is a development artifact and should be removed, or if it's meant to save both the source map and the main bundle to the local file system -- if that's the case, maybe a more general name the deemphasizes the source-map ('build'?), and changing it to `webpack --config webpack.config.js`.

Comment (trevorackerman) [inline package.json:23]: The sourcemap npm script itself is really just for creating the sourcemap artifact so it's easy to debug the trailsy code in the browser console.

I thought it was a little challenging at first to effectively debug trailsy in the browser console.
I and others weren't initially aware how to generate a source map and debug individual js files as opposed to bundle.js

The readme file should be updated to say what the sourcemap is for and how to use it to debug the application in the browser console.

As a bigger conversation for Wednesday we could talk about:
How do we best make the project to make it easy to understand and debug the application?

Do we need to update our readme file more? Should we add github pages to the project?

I like how nice and  polished the generated github pages look.
It's easy to do and then we can just put a link to them in the readme markdown file.

If you click on the settings for the trailsy project
settings

And then click on the button to launch the page generator
launch

Then you can end up with a starting point like this:
http://trevorackerman.github.io/trailsy/

**Comment by garborg** [comment]
Latest commits look great. Merging ahead of tonight's session. We can address any unaddressed comments in session.

**Pull request merged**